packages/esm-ward-app/src/config-schema.ts

```diff
@@ -6,16 +6,17 @@ const defaultWardPatientCard: WardPatientCardDefinition = {
   rows: [
     {
       rowType: 'header',
-      elements: ['bed-number', 'patient-name', 'patient-age', 'patient-address'],
+      elements: ['bed-number', 'patient-name', 'patient-age', 'patient-address', 'patient-identifier'],
     },
   ],
   appliedTo: null,
 };
 
 const defaultPatientAddressFields: Array<keyof PersonAddress> = ['cityVillage', 'country'];
-
+const defaultIdentifierTypeUuid = '05a29f94-c0ed-11e2-94be-8c13b969e334';//OpenMrs ID
```

Comment:
No, please get rid of this.  Do not assume any specific metadata.

Reply:
Made the change. Thanks @mseaton .